workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/utils/TicketPayloadMapper.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {TicketPayload} from '../types';
+
+export function TicketPayloadMapper(ticket: TicketPayload) {
+	let suggestions = [];
+
+	try {
+		suggestions = JSON.parse(ticket?.suggestions);
+	}
+	catch (error) {}
+
+	delete ticket.actions;
```

Comment:
Since we are returning an entirely new object, why do we need to delete ticket.actions?

Reply:
Hey @ethib137 ,
We are using the payload to call Liferay APIs.
TicketPayload : this is the object thats being sent and received to/from the server .. if I send the same object including actions, it fails.. that's why I'm deleting it.
Ticket: this is the object we are using to bind to the UI - if this is sent to the APIs, it will cause 500 error.